Tests/Adapter/SymfonyCacheTest.php

```diff
@@ -111,4 +111,147 @@ public function testCacheActionWithInvalidType()
 
         $this->cache->cacheAction('token', 'invalid-type');
     }
+
+    /**
+     * Asserts the flush method throws an exception if the version IP of the server cannot be detected.
+     *
+     * @expectedException        \InvalidArgumentException
+     * @expectedExceptionMessage "wrong ip" is not a valid ip address
```

Comment:
Please use `->setExceptedException()` instead, right before the line that should trigger the exception.